coredump_uploader/__init__.py

```diff
@@ -198,6 +198,7 @@ def to_json(self):
 
 
 def code_id_to_debug_id(code_id):
+    code_id += "00"*16
```

Comment:
id as extracted can be as little as 8 bytes, extend it.

Reply:
IIRC it is fine to have a short CodeID, it is the DebugID which has a more standard-length format (guid + age).  So I don't think this is necessary.  Is there a problem with the short CodeIDs?

Reply:
Yes, the UUID() call breaks if there are to few bytes.  This is to pad them with zero.